packages/test/functional-tests/README.md

```diff
@@ -1 +1,46 @@
 # @fluid-internal/functional-tests
+
+TODO: Document this package
```

Comment:
Not for this PR, but we shouldn't have packages (even private ones) without basic documentation explaining what they are.